### PR TITLE
Initial version

### PR DESCRIPTION
**Pull request by TommyJuuti**
Original implementation:
[https://github.com/marketplace/actions/check-version](https://github.com/marketplace/actions/check-version)

Changes made:
- Fix PR base detection
  * base was looked from base.sha, which doesn't seem to exist in our PR details. Use base.ref instead
- Replace version compare with semver function
  * original implementation failed when major updated but minor or patch reverted to lower value (0.9.0 -> 1.0.0)
- Handle also initial commit, when no package json in master yet
- Add hook and npm script to build action
- Simplify inputs and outputs

Tested with iot-asset-web-app ([PR#110](https://github.com/SADEInnovations/iot-asset-web-app/pull/110))

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,36 @@
...
+## build
+
+Checker is automatically built with pre-commit hook, but if needed, manual build is possible: </br>
+`npm run build`
+
+## usage
+
+```
+- name: Check version
+  uses: SADEInnovations/version-checker@initial
```
Comment (ujoni) [inline README.md]: The `@initial` notations can be removed from the examples as this goes to master

File: package.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
+{
+  "name": "version-checker",
+  "version": "0.2.1",
+  "description": "NPM version checker for pull requests",
+  "author": "SADE Innovations Oy",
+  "license": "MIT",
+  "private": "false",
+  "main": "index.js",
```
Comment (ujoni) [inline package.json]: `dist/index.js` or `version-checker.js`

**Review by ujoni — CHANGES_REQUESTED** [review]
Some small issues

File: package.json (hunk 2 above)
Comment (TommyJuuti) [reply package.json]: `dist/index.js`

File: README.md (hunk 1 above)
Comment (TommyJuuti) [reply README.md]: Good point 👍

**Comment by TommyJuuti** [comment]
Fixes made

**Review by ujoni — APPROVED** [review]
:+1:

**Pull request merged**